packages/react-components/source/scss/library/components/_tabs.scss

```diff
@@ -8,30 +11,68 @@
   max-width: 100%;
 }
 
+.rc-tabs-list-container {
+  align-items: flex-end;
+  display: flex;
+}
+
 .rc-tabs-list {
   align-items: flex-end;
+  box-shadow: $bottom-border-shadow;
```

Comment:
This PR changes the border at the top of the tab panel area so that it looks the same as before, but is rendered as a box shadow on the bottom of the tab bar instead of as border on the top of the panel. I couldn't find a good combination of z-index values that would let all the focus styles, tab buttons, and scroll buttons layer correctly for both the transparent and default designs using the border approach.

Reply:
yea this is great, I had to use the same implementation in relay-ui